knowledge-base/actions/actionsdesk/lfs-warning/action-security.yml

```diff
@@ -0,0 +1,15 @@
+name: "LFS-warning"
+github-token:
+  action-input:
+    input: token
+    is-default: false
```

Comment:
is-default: true

Reference: https://github.com/ActionsDesk/lfs-warning/blob/e5f9a4c21f4bee104db7c0f23954dde59e5df909/action.yml#L7

Reply:
@RksRocks